packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -26,6 +30,38 @@ export default function callApi({
     signal,
   };
 
+  if (method === 'GET' && CACHE_AVAILABLE) {
```

Comment:
@kristw and I were discussing whether this should be opt-in or not. currently there's no way to opt-out, though I guess if if depends on the server/backend implementing `Etag`s then maybe that's a non-issue.

any thoughts on this?

Reply:
I could add a new option to `callApi`, to turn this on/off. In the initial tests I did with dashboards and ETag, some of the dashboards load in half the time, so I think we might want to default this to be enabled.

Reply:
I have merged and added this new issue for turning conditional request on/off.
https://github.com/apache-superset/superset-ui/issues/128